<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>359</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>登陆</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ICO/主题.png</normaloff>ICO/主题.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
        border:1px solid gray;
        border-radius:10px;
        padding:2px 4px;
		background-color: rgba(255, 255, 255, 0.5);
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="denglu">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文彩云</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>欢迎使用飞云信息管理平台</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="mima">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>123</y>
       <width>201</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
        border:1px solid gray;
        border-radius:10px;
        padding:2px 4px;
		background-color: rgba(255, 255, 255, 0.5);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>ICO/png/shell32.dll(45).png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="zhanghao">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>74</y>
       <width>201</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
        border:1px solid gray;
        border-radius:10px;
        padding:2px 4px;
		background-color: rgba(255, 255, 255, 0.5);
}</string>
     </property>
     <property name="placeholderText">
      <string>工号</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap>ICO/png/shell32.dll(269).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="guanbi">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>180</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border:1px solid gray;
        color:black;
        font-size:16;
		border-radius:10px;
        padding-left:5px;
        padding-right:10px;
        text-align:middle;
		background:LightGray;
		background-color: rgba(255, 255, 255, 0.5);
    }
    QPushButton:hover{
        color:white;
        border:1px solid #40E0D0;
        border-radius:10px;
        background:#40E0D0;
    }</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shezhi_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border:1px solid gray;
        color:black;
        font-size:11;
		border-radius:10px;
        padding-left:5px;
        padding-right:10px;
        text-align:middle;
		background:LightGray;
		background-color: rgba(255, 255, 255, 0.5);
    }
    QPushButton:hover{
        color:white;
        border:1px solid #40E0D0;
        border-radius:10px;
        background:#40E0D0;
    }</string>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="denglu_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border:1px solid gray;
        color:black;
        font-size:11;
		border-radius:10px;
        padding-left:5px;
        padding-right:10px;
        text-align:middle;
		background:LightGray;
		background-color: rgba(255, 255, 255, 0.5);
    }
    QPushButton:hover{
        color:white;
        border:1px solid #40E0D0;
        border-radius:10px;
        background:#40E0D0;
    }</string>
     </property>
     <property name="text">
      <string>登陆</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="shezhi">
    <attribute name="title">
     <string>设置</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>311</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="queren">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
        border:1px solid gray;
        color:black;
        font-size:11;
		border-radius:10px;
        padding-left:5px;
        padding-right:10px;
        text-align:middle;
		background:LightGray;
		background-color: rgba(255, 255, 255, 0.5);
    }
    QPushButton:hover{
        color:white;
        border:1px solid #40E0D0;
        border-radius:10px;
        background:#40E0D0;
    }</string>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="chongzhi">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
        border:1px solid gray;
        color:black;
        font-size:11;
		border-radius:10px;
        padding-left:5px;
        padding-right:10px;
        text-align:middle;
		background:LightGray;
		background-color: rgba(255, 255, 255, 0.5);
    }
    QPushButton:hover{
        color:white;
        border:1px solid #40E0D0;
        border-radius:10px;
        background:#40E0D0;
    }</string>
        </property>
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="tuichu">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
        border:1px solid gray;
        color:black;
        font-size:11;
		border-radius:10px;
        padding-left:5px;
        padding-right:10px;
        text-align:middle;
		background:LightGray;
		background-color: rgba(255, 255, 255, 0.5);
    }
    QPushButton:hover{
        color:white;
        border:1px solid #40E0D0;
        border-radius:10px;
        background:#40E0D0;
    }</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>服务器ID：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="fuwuqiID">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="inputMask">
           <string>999.999.999.999</string>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数据库名：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="shujukuming">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>mysql</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>服务账号：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="fuwuqizhanghao">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>root</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>服务密码：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="fuwuqimima">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>080420</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="cursorPosition">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>服务端口：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="fuwuduankou">
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3306</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>zhanghao</tabstop>
  <tabstop>mima</tabstop>
  <tabstop>denglu_2</tabstop>
  <tabstop>guanbi</tabstop>
  <tabstop>fuwuqiID</tabstop>
  <tabstop>shujukuming</tabstop>
  <tabstop>fuwuqizhanghao</tabstop>
  <tabstop>fuwuqimima</tabstop>
  <tabstop>fuwuduankou</tabstop>
  <tabstop>queren</tabstop>
  <tabstop>chongzhi</tabstop>
  <tabstop>tuichu</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>guanbi</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tuichu</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
